tutorials/using-xgboost/src/run.py

```diff
@@ -0,0 +1,73 @@
+# imports
+import mlflow
+import argparse
+
+import pandas as pd
+import xgboost as xgb
+import dask.dataframe as dd
+
+from distributed import Client
+from dask_mpi import initialize
+from adlfs import AzureBlobFileSystem
+
+# argparse setup
+parser = argparse.ArgumentParser()
```

Comment:
This logic should go under an `if __name__ == "__main__"` block.

Reply:
given the discussion, should we do this for this example?

Reply:
For a file this large (70+ lines) I stand by the comment (aka: yes).